ql/ql/src/queries/style/AndroidIdPrefix.ql

```diff
@@ -11,6 +11,8 @@ import ql
 
 /** Holds if `t` transitively imports an Android module. */
 predicate importsAndroidModule(TopLevel t) {
+  t.getFile() =
+    any(YAML::QLPack pack | pack.getExtractor() = "java").getADependency*().getAFileInPack() and
```

Comment:
```suggestion
    any(QLPack pack | pack.getADependency*().getExtractor() = "java").getAFileInPack() and
```

I accidentally flipped the direction of `getADependency*()`.  

I ninja edited my comment, but you were fast enough to grap the incorrect version.

Reply:
😅 Should have verified it myself instead of blindly copy-pasting, but since I can't run this locally I was expecting the CI to tell me whether it was correct...